src/Psl/Xml/Error/XmlErrorCollection.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Xml\Error;
+
+use Countable;
+use IteratorAggregate;
+use Psl\Arr;
+use Psl\Iter;
+use Psl\Iter\Iterator;
+use Psl\Str;
+
+use const Psl\Xml\ERR_WARNING;
+
+final class XmlErrorCollection implements Countable, IteratorAggregate
```

Comment:
```suggestion
final class ErrorCollection implements Countable, IteratorAggregate
```

we don't have to repeat ourselves :) Xml is already in the FQCN.